app/mailers/club_application_mailer.rb

```diff
@@ -12,9 +12,9 @@ def applicant_confirmation(application)
   def admin_notification(application)
     @application = application
 
-    to = Mail::Address.new 'team@hackedu.us'
+    to = Mail::Address.new 'team@hackclub.io'
     to.display_name = 'Zach Latta'
     mail(to: to.format, from: @application.mail_address.format,
-         subject: 'hackEDU Application')
+         subject: 'Hack Club Application')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.